api/src/http.rs

```diff
@@ -404,6 +404,9 @@ pub struct ProxyConfig {
     pub fallback: bool,
     /// Interval of P2P proxy health checking, in seconds.
     pub check_interval: u64,
+    /// Replace url sent to proxy from https to http. So no CA is needed to be passed to the proxy.
```

Comment:
We can't assume the proxy server behavior, so I think this comment is better:

```
Replace URL to http to request source registry with proxy, and allow fallback to https if the proxy is unhealthy.
```

Reply:
Yes it's better

Reply:
replaced